src/server/game/Battlegrounds/ArenaTeam.cpp

```diff
@@ -658,7 +658,7 @@ uint32 ArenaTeam::GetPoints(uint32 memberRating)
 
     if (rating <= 1500)
     {
-        if (sWorld->getIntConfig(CONFIG_ARENA_SEASON_ID) < 6)
+        if (sWorld->getIntConfig(CONFIG_ARENA_SEASON_ID) < 6 && sWorld->getIntConfig(CONFIG_LEGACY_ARENA_POINTS_CALC) == 1)
```

Comment:
```suggestion
        if (sWorld->getIntConfig(CONFIG_ARENA_SEASON_ID) < 6 && sWorld->getIntConfig(CONFIG_LEGACY_ARENA_POINTS_CALC))
```

if statements are booleans, so if sWorld->getIntConfig() returns anything but 0, it will be true

Reply:
We play that var is the getIntConfig(CONFIG_LEGACY_ARENA_POINTS_CALC) in this case

if (var) is the same as if (var == 1)

and

if (!var) is the same as if (var == 0)

Reply:
OK, I see!